cdap-ui/app/hydrator/controllers/create/toppanel-ctrl.js

```diff
@@ -212,6 +242,7 @@ class HydratorPlusPlusTopPanelCtrl {
     this.metadataExpanded = false;
     event.preventDefault();
     event.stopPropagation();
+    this.onSaveDraft();
```

Comment:
Other than name, what else can trigger a `saveMetadata` event?

Reply:
Right now changing name and description (part of metadata section) triggers the save. Other than that the user can click on "Save" on the top panel to save the pipeline as draft.